packages/token/src/erc20/utils.cairo

```diff
@@ -0,0 +1,32 @@
+use openzeppelin_token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTrait};
+use starknet::ContractAddress;
+
+mod Errors {
+    pub const TRANSFER_FAILED: felt252 = 'ERC20 transfer failed';
+}
+
+/// A module containing utility functions for interacting with ERC-20 token contracts.
+#[generate_trait]
+pub impl ERC20Utils of ERC20UtilsTrait {
```

Comment:
I understand your concerns and would like to also highlight benefits of this approach:
- Helper fns for transfers make sure you **never** forget to check the return value
- It does introduce a new utils module, but so does the Solidity library with the [SafeERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/utils/SafeERC20.sol)
- It can and should become a standard way to make a transfer because it's more secure and concise

Reply:
@andrew-fleming It would be interesting to hear your thoughts on that matter

Reply:
That should most probably be a different issue, not part of the Vesting component implementation.